ui/demoBot.html

```diff
@@ -84,8 +84,10 @@
 
 
     var botcoord;
-    postMove('http://localhost:5000/prediction', coord,
-      function(data) {botcoord = JSON.parse(data)
+    var port = window.location.port;
+    var host = window.location.host;
+    postMove("//" + host + (port=="8081" ? "" : ":8081") + "/prediction", coord,
```

Comment:
I think it's fine to require that, since you added support for serving the static files from the demo web server in this same pull request.

Generally it will be easier if the JS and the bot are served on the same host anyways, cause then we will avoid having to deal with cross-origin requests.